lms/djangoapps/courseware/user_state_client.py

```diff
@@ -53,72 +68,6 @@ def __init__(self, user=None):
         """
         self.user = user
 
-    @contract(
-        username="basestring",
-        block_key=UsageKey,
-        scope=ScopeBase,
-        fields="seq(basestring)|set(basestring)|None"
-    )
-    def get(self, username, block_key, scope=Scope.user_state, fields=None):
-        """
-        Retrieve the stored XBlock state for a single xblock usage.
-
-        Arguments:
-            username: The name of the user whose state should be retrieved
-            block_key (UsageKey): The UsageKey identifying which xblock state to load.
-            scope (Scope): The scope to load data from
-            fields: A list of field values to retrieve. If None, retrieve all stored fields.
-
-        Returns:
-            dict: A dictionary mapping field names to values
-
-        Raises:
-            DoesNotExist if no entry is found.
-        """
-        try:
-            _usage_key, state = next(self.get_many(username, [block_key], scope, fields=fields))
-        except StopIteration:
-            raise self.DoesNotExist()
-
-        return state
-
-    @contract(username="basestring", block_key=UsageKey, state="dict(basestring: *)", scope=ScopeBase)
-    def set(self, username, block_key, state, scope=Scope.user_state):
-        """
-        Set fields for a particular XBlock.
-
-        Arguments:
-            username: The name of the user whose state should be retrieved
-            block_key (UsageKey): The UsageKey identifying which xblock state to update.
-            state (dict): A dictionary mapping field names to values
-            scope (Scope): The scope to load data from
-        """
-        self.set_many(username, {block_key: state}, scope)
-
-    @contract(
-        username="basestring",
-        block_key=UsageKey,
-        scope=ScopeBase,
-        fields="seq(basestring)|set(basestring)|None"
-    )
-    def delete(self, username, block_key, scope=Scope.user_state, fields=None):
-        """
-        Delete the stored XBlock state for a single xblock usage.
-
-        Arguments:
-            username: The name of the user whose state should be deleted
-            block_key (UsageKey): The UsageKey identifying which xblock state to delete.
-            scope (Scope): The scope to delete data from
-            fields: A list of fields to delete. If None, delete all stored fields.
-        """
-        return self.delete_many(username, [block_key], scope, fields=fields)
-
-    @contract(
-        username="basestring",
-        block_key=UsageKey,
-        scope=ScopeBase,
-        fields="seq(basestring)|set(basestring)|None"
-    )
     def get_mod_date(self, username, block_key, scope=Scope.user_state, fields=None):
```

Comment:
Is this getting wiped in another PR? (I'm also happy to do that if you'd like -- just thought it was going in this one.)